spec/support/db_information_reloader.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2020, Stiftung für junge Auslandssschweizer. This file is part of
+#  hitobito_sjas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sjas.
+
+Person.reset_column_information
```

Comment:
Können wir das gleich noch generisch im Core machen? Oder möchtest du lieber dem Problem mal auf den Grund gehen?

Reply:
Beides :-)

ich schau mal, ob ich da ne guten Lösung im Core finde, die dann auch in den Wagons funktioniert.